mesonbuild/compilers/detect.py

```diff
@@ -606,6 +617,37 @@ def sanitize(p: str) -> str:
             return cls(
                 ccache, compiler, compiler_version, for_machine, is_cross, info,
                 exe_wrap, full_version=full_version, linker=linker)
+        if 'TASKING VX-toolset' in err:
+            if 'TriCore' in err:
+                cls = c.TaskingTricoreCCompiler
+                lnk = linkers.TaskingTricoreLinker
+            elif 'ARM' in err:
+                cls = c.TaskingArmCCompiler
+                lnk = linkers.TaskingARMLinker
+            elif '8051' in err:
+                cls = c.Tasking8051CCompiler
+                lnk = linkers.Tasking8051Linker
+            elif 'PCP' in err:
+                cls = c.TaskingPCPCCompiler
+                lnk = linkers.TaskingPCPLinker
+            elif 'MCS' in err:
+                cls = c.TaskingMCSCCompiler
+                lnk = linkers.TaskingMCSLinker
+            else:
+                raise EnvironmentException('Failed to detect linker for TASKING VX-toolset compiler. Please update your cross file(s).')
+
+            tasking_ver_match = re.search(r'v(\d+)\.(\d+)r(\d+) Build (\d+)', err)
+            assert tasking_ver_match is not None, 'for mypy'
+            tasking_version = '.'.join(x for x in tasking_ver_match.groups() if x is not None)
+
+            env.coredata.add_lang_args(cls.language, cls, for_machine, env)
+            ld = env.lookup_binary_entry(for_machine, cls.language + '_ld')
+
+            linker = lnk(ld, for_machine, version=tasking_version) # Fix needed here
+            return cls(
+                ccache, compiler, tasking_version, for_machine, is_cross, info,
+                exe_wrap, full_version=full_version, linker=linker)
+        # TODO add detection logic here
```

Comment:
Will you do it?

Reply:
I think I have left that there by mistake, I'll take that comment out.